go/engine/signup_test.go

```diff
@@ -18,10 +18,20 @@ func AssertDeviceID(g *libkb.GlobalContext) (err error) {
 }
 
 func TestSignupEngine(t *testing.T) {
+	subTestSignupEngine(t, false)
+}
+
+func TestSignupEngineSDH(t *testing.T) {
+	subTestSignupEngine(t, true)
```

Comment:
Yeah it does [SkipPaper](https://github.com/keybase/client/blob/master/go/engine/common_test.go#L92)